### DISCUSSION TITLE
subfolder routing

### DISCUSSION THREAD

**Discussion by Naveed-Jameel** [vercel/next.js#40873] [Help] [answered]
i want to navigate between subfolder file but routing does not work .
pages
    product
         file1
         file2
         file3

<Link href={"/product/file1"}>go to file 1</Link>
<Link href={"/product/file2"}>go to file 2</Link>
<Link href={"/product/file3"}>go to file 3</Link>

first time when i click on Link its fine but then from inside my file1 when i click on  Link it give
404 error  "localhost:300/product/product/file2"   what is the problem?

**Comment by gaston-flores** [comment, marked as answer]
Are you linking to `href="product/file1"`, without an absolute path? If so, you might want to change that to `href="/product/file1"` 

Can you share a snippet of how you are linking between pages?

**Reply by Naveed-Jameel** [reply]
@gaston-flores Thank You it works
